### PR TITLE
removed library from the .NET Standard TOC entry

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

**Review by guardrex — APPROVED** [review]

File: docs/standard/toc.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # [Get Started with .NET](get-started.md)
 # [Tour of .NET](tour.md)
 # [.NET Architectural Components](components.md)
-# [.NET Standard Library](library.md)
```
Comment (svick) [inline docs/standard/toc.md]: Should the article URL/file name change too?

I think it's kind of weird that the ".NET Standard" article has the URL https://docs.microsoft.com/en-us/dotnet/articles/standard/library.

Comment (mairaw) [inline docs/standard/toc.md]: Yeah, I've been wondering whether I should do that or not. I'll change it to net-standard.

**Comment by mairaw** [comment]
Another option for the .NET Standard article would be to change its title to .NET Standard overview and then change the path to dotnet/standard/overview. Thoughts @BillWagner @rpetrusha @svick @GuardRex?

**Comment by GuardRex** [comment]
I worry that it isn't an "overview" relative to some other more detailed topics on the subject. I expect an "overview" to provide general, high-level content before I access detailed, low-level content. In this case, the topic doesn't seem to be an "overview" relative to other topics on the same subject matter.

I like ... **dotnet/net-platform-guide/net-standard**

Yes, that's just on my wish list: "net-platform-guide" over "standard," "net-core-guide" over "core," and "net-framework-guide" over "framework."

**Review by BillWagner — APPROVED** [review]
LGTM.

I did a search on the master branch, and I think you've found all instances of "library".

Let's :shipit:

**Pull request merged**